### PR TITLE
Initial 3VL work

### PR DESCRIPTION
**Pull request by aminusfu**

### CODE REVIEW

File: source/feature-guide/feature-deriving.rst [hunk 1]
```diff
@@ -616,6 +441,159 @@ ZCL function like:
...
+
+What can you do with logical variables? A few things:
+
+-  You can use them as filters. Any rows where the logical variable
+   is "selected" match the filter and are retained; any that are
+   "other" or "missing" do not match and are filtered out.
+-  You can reduce them to just three values with the ``as_selected``
+   function. This lets you:
+
+   +  Get "the right numbers" for Multiple Response analyses by
```
Comment (jonkeane) [inline source/feature-guide/feature-deriving.rst]: I would change 'Get "the right numbers" for ...' to 'Calculate proportions and other statistics accurately for...'

File: source/feature-guide/feature-deriving.rst [hunk 2]
```diff
@@ -624,6 +449,159 @@ ZCL function like:
...
+too: SQL implements ``NULL`` in addition to ``TRUE`` and ``FALSE``,
+and general languages often have a similar ``None`` or ``NA`` value.
+
+In Crunch, rather than having a "boolean" type with true/false/null
+values, we instead allow *any* categorical variable to participate
+in logic. Categories which have the ``selected: true`` flag are
+"selected". Categories which have the ``missing: true`` flag are
+"missing", and any categories that lack both are "other".
+Any categorical variable that has at least one of each can function
+as a "logical variable".
```
Comment (dhandy2013) [inline source/feature-guide/feature-deriving.rst]: Maybe you should make it even more crystal clear that to be used as a logical variable, a categorical variable needs at least three categories: at least one selected category, at least one missing category, and at least one "other" category.

Comment (dhandy2013) [reply source/feature-guide/feature-deriving.rst]: I'm concerned that someone might read "at least one of each" as "at least one selected and one missing".

Comment (aminusfu) [reply source/feature-guide/feature-deriving.rst]: That was actually wrong; fixed now.

**Comment by dhandy2013** [comment]
LGTM

**Pull request merged**